src/components/TodoList/TodoList.tsx

```diff
@@ -21,11 +22,18 @@ import { TransitionGroup } from 'react-transition-group';
 
 const TodoList = () => {
   const todos = useSelector((state: RootState) => state.todosList.todos);
+  const status = useSelector((state: RootState) => state.todosList.status);
 
-  const dispatch = useDispatch();
+  const sortedTodos = useMemo(() => {
+    return todos.slice().sort((a:Todo, b:Todo) => b.timestamp.getTime() - a.timestamp.getTime());
```

Comment:
Why `slice`?

Reply:
The slice() method creates a shallow copy of the 'todos' array. This is necessary when using useMemo because its arguments might be read-only, and direct modifications (like sorting) would result in an error.